src/applications/coronavirus-chatbot/index.js

```diff
@@ -153,6 +153,6 @@ const chatRequested = scenario => {
 
 export default function initializeChatbot(_root) {
   root = _root;
-  watchForButtonClicks();
+  watchForButtonClicks(root);
```

Comment:
Looks like this is vestigial from testing? `watchForButtonClicks` doesn't take an argument.

Reply:
Yes, will remove root